src/main/pages/che-7/end-user-guide/assembly_network-troubleshooting.adoc

```diff
@@ -0,0 +1,30 @@
+---
+title: Network Troubleshooting
+keywords:
+tags: []
+sidebar: che_7_docs
+permalink: che-7/network-troubleshooting/
+folder: che-7/end-user-guide
+summary:
+---
+
+:page-liquid:
+:parent-context-of-network-troubleshooting: {context}
+
+[id="network-troubleshooting"]
+= Network troubleshooting
+
+:context: network-troubleshooting
+
+The following section describes how to prevent or solve issues related to corporate network policies. The network administrator may be required to enable ports or WebSocket, which Che requires for proper functionality.
```

Comment:
Why 'corporate'?

Reply:
Originally, I had `the customer's company network`. I was suggested by Sun that I should use the word `corporate` instead.

Reply:
What we are trying to troubleshoot here is problems related to network policies and rules defined by the IT (or sub division) of a company. `Corporate network policies` sounds good.